Sources/DebugDescriptions.swift

```diff
@@ -24,6 +25,15 @@ private enum Element {
     case sections([SectionViewModel])
     case registrations(Set<ViewRegistration>)
     case isEmpty(Bool)
+    case diffOnBackgroundQueue(Bool)
+    case reloadDataOnReplacingViewModel(Bool)
```

Comment:
Is it possible to consolidate these to be something like:

```swift
case bool(label: String, value: Bool)
```

Then we can remove a lot of duplication.

Reply:
Done.